app/models/project.rb

```diff
@@ -40,6 +40,8 @@ class Project < ActiveRecord::Base
   has_many :tagged_resources, as: :resource
   has_many :tags, through: :tagged_resources
 
+  enum state: [:paused, :finished]
```

Comment:
Does this conflict with the current finished method [here](https://github.com/zooniverse/Panoptes/blob/aa00e646aaa458716220dd2d8fd3542bebb4bc28/app/models/project.rb#L139)?